components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -1111,6 +1115,89 @@
+        logger.info(f"No retention period is configured, skipping {component_name}...")
+        return
+
+    logger.info(f"Starting {component_name}...")
+
+    container_name = f"clp-{component_name}-{instance_id}"
+    if container_exists(container_name):
+        return
+
+    container_config_filename = f"{container_name}.yml"
+    container_config_file_path = clp_config.logs_directory / container_config_filename
+    with open(container_config_file_path, "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)